packages/model/src/research-output.ts

```diff
@@ -218,6 +218,12 @@ export type ResearchOutputDataObject = ResearchOutputCoreObject & {
   subtype?: string;
   teams: Pick<TeamResponse, 'id' | 'displayName'>[];
   workingGroups: Pick<WorkingGroupResponse, 'id' | 'title'>[];
+  relatedResearch?: Array<
```

Comment:
```suggestion
  relatedResearch: Array<
```

I believe we can remove this `?` as in the parser (https://github.com/yldio/asap-hub/blob/crn-1298-related-research-to-shared-output/apps/crn-server/src/entities/research-output.ts#L67:L70) we guarantee that `relatedResearch` is always returned, right?